### DISCUSSION TITLE
トラックボールが動作しない、はんだ部分の変色

### DISCUSSION THREAD

**Discussion by pythonista-blitz** [Yowkees/keyball#330] [Q&A] [answered]
トラックボールが動かなくなった現象が発生し、気になるジャンパ部分のはんだを付け直していたのですが、写真の通り変色してはんだが乗らなくなってしまいました。
![IMG_0579](https://github.com/Yowkees/keyball/assets/23119416/6e34707d-7a2e-4052-8324-6c838728c81d)

これらの問題について質問させていただきたいのですが、基盤の交換が必要なのか、他に解決策があるのかについてアドバイスをいただけますか？
一応他の部分もはんだ部分の画像も添付しておきます。
![IMG_0580](https://github.com/Yowkees/keyball/assets/23119416/2da3b294-41ae-4fbb-b9b2-c91cce431444)
![IMG_0582](https://github.com/Yowkees/keyball/assets/23119416/bfdf4c64-9c39-4355-80de-bf7fcf5017e2)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

１枚目写真のジャンパはOLED用のジャンパでトラックボールの動きには関係ありません。
変色したジャンパは諦め、下記画像の緑点２点をエナメル線やゴム被覆銅線で接続していただくと、ジャンパしたのと同じ回路となり、OLED復活すると思います。

![KB44R_OLEDfix](https://github.com/Yowkees/keyball/assets/84209772/2a855b4f-1ba7-47bd-917b-00012f24b4e6)

次にトラックボールが動かない件ですが、半田付けや部品の向き等は大丈夫そうですので、L字7Pコンスルーをご確認ください。
下記画像のように端子に変形があると、導通不良となります。ピン形状が大丈夫かご確認いただき、変形があればピンセット等で成形してください。
また、トラックボール基板が奥まで差し込まれているかもご確認お願いします。

![2_conthru_henkei](https://github.com/Yowkees/keyball/assets/84209772/9c473183-0a68-4012-bccf-faf405bc3e7b)

以上、お手数ですがご確認お願いいたします。

**Comment by pythonista-blitz** [comment]
おっしゃる通りL字コンスルーの接触不良でした。無事元通り動いています。
ありがとうございました！
